### ISSUE TITLE
Pretrained resnet-18 and resnet-34 give nan error during training

### ISSUE THREAD

**Issue by jessika0991** [facebook/fb.resnet.torch#125]
Hello,

I am having problems training two of the pretrained models (18 and 34). When it starts training, the error value keeps increasing until getting value inf or nan. There is something wrong with these two models? Or am I doing something wrong?

Thanks

**Comment by colesbury** [comment]
Sounds like you might have a learning rate issue. The learning rate schedule is set for batch size 256 and training from scratch. If you use a smaller batch size, you could try decreasing the learning rate proportionally. You might also want to use a smaller learning rate if you are fine-tuning a pretrained model. (Maybe 0.01 instead of 0.1 to start decay more quickly in [Trainer:learningRate](https://github.com/facebook/fb.resnet.torch/blob/f7689c709cd6193a250ea7171356299d40ee459a/train.lua#L164-L175))

**Comment by jessika0991** [comment]
Hi,

I forgot to let you know that your suggestion worked. Thanks again
